src/providers/cortex/trulens/providers/cortex/endpoint.py

```diff
@@ -14,9 +15,51 @@
 pp = pprint.PrettyPrinter()
 
 
+# the credit consumption table needs to be kept up-to-date with
+# the latest cost information https://www.snowflake.com/legal-files/CreditConsumptionTable.pdf#page=9.
+CORTEX_MODEL_TO_CREDITS_PER_1M_TOKENS = {
```

Comment:
how often do you expect this to change?

Reply:
I hope very infrequently (like no more than once a quarter).

With that being said, I've flagged this to the PM (Harshal) of  Cortex LLM functions team and hopefully can get them on releasing a REST API for cost tracking after LLM Function's PuPr